### PR TITLE
Language and minimalism authenticating-with-openshift-connector-from-che

### PR DESCRIPTION
**Pull request by themr0c**
Language and minimalism on proc_authenticating-with-openshift-connector-from-che.adoc

> Read our [Contribution guide](https://github.com/eclipse/che-docs/blob/master/CONTRIBUTING.adoc) before submitting a PR.

### What does this PR do?

* Fix vale errors (except 10 suggestions).
* Compliance with minimalism.

### PR Checklist

As the author of this Pull Request I made sure that:

- [x] `vale` has been run successfully against the PR branch
- [x] Link checker has been run successfully against the PR branch
- [x] Documentation describes a scenario that is already covered by QE tests, otherwise an issue has been created and acknowledged by Che QE team
- [x] Changed article references are updated where they are used (or a redirect has been set up on the docs side):
    - [x] Dashboard [branding.json](https://github.com/eclipse/che-dashboard/blob/master/src/components/branding/branding.json)
    - [x] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)

### CODE REVIEW

**Comment by themr0c** [comment]
Put in a separate request additional work made for https://github.com/skabashnyuk/che-docs/pull/2.

File: modules/extensions/partials/proc_authenticating-with-openshift-connector-from-che.adoc [hunk 1]
```diff
@@ -1,53 +1,63 @@
 // using-openshift-connector-in-eclipse-che
 
-
-
 [id="authenticating-with-openshift-connector-from-{prod-id-short}_{context}"]
-= Authenticating with OpenShift Connector from {prod-short}
+= Authenticating with OpenShift Connector from {prod-short} when the OpenShift OAuth service does not authenticate the {prod-short} instance
 
-The following section is relevant only when the OpenShift OAuth service does not already authenticate a {prod-short} instance; otherwise, the OpenShift Connector plug-in automatically establishes the authentication with the Openshift instance where {prod-short} runs.
+This section describes how to authenticate with an OpenShift cluster when the OpenShift OAuth service does not authenticate the {prod-short} instance. It enables the user to develop and push Components from {prod-short} to the OpenShift instance containing {prod-short}.
```
Comment (yhontyk) [inline modules/extensions/partials/proc_authenticating-with-openshift-connector-from-che.adoc]:
```suggestion
This section describes how to authenticate with an OpenShift cluster when the OpenShift OAuth service does not authenticate the {prod-short} instance. It enables the user to develop and push Components from {prod-short} to the OpenShift instance that contains {prod-short}.
```

Comment (yhontyk) [reply modules/extensions/partials/proc_authenticating-with-openshift-connector-from-che.adoc]: Nothing wrong with it, just avoiding ambiguity gerund introduces

File: modules/extensions/partials/proc_authenticating-with-openshift-connector-from-che.adoc [hunk 2]
```diff
@@ -1,53 +1,63 @@
...
 
-Before the user can develop and push Components from {prod-short}, they need to authenticate with an OpenShift cluster.
+[NOTE]
+====
+When the OpenShift OAuth service does authenticate the {prod-short} instance, the OpenShift Connector plug-in automatically establishes the authentication with the OpenShift instance containing {prod-short}.
```
Comment (yhontyk) [inline modules/extensions/partials/proc_authenticating-with-openshift-connector-from-che.adoc]:
```suggestion
When the OpenShift OAuth service authenticates the {prod-short} instance, the OpenShift Connector plug-in automatically establishes the authentication with the OpenShift instance containing {prod-short}.
```

File: modules/extensions/partials/proc_authenticating-with-openshift-connector-from-che.adoc [hunk 3]
```diff
@@ -1,53 +1,63 @@
...
+====
 
 OpenShift Connector offers the following methods for logging in to the OpenShift Cluster from the {prod-short} instance:
 
-* Using the notification that asks to log in to the OpenShift cluster where {prod-short} is deployed to.
+* Using the notification asking to log in to the OpenShift instance containing {prod-short}.
```
Comment (yhontyk) [inline modules/extensions/partials/proc_authenticating-with-openshift-connector-from-che.adoc:15]: Not sure why we need gerunds here. IMO "that asks" is less ambiguous than the gerund.

File: modules/extensions/partials/proc_authenticating-with-openshift-connector-from-che.adoc [hunk 4]
```diff
@@ -1,53 +1,63 @@
...
 * Using the btn:[Log in to the cluster] button.
 * Using the Command Palette.
 
 [NOTE]
 ====
-.In {prod-short} {prod-ver}, Openshift Connector plug-in requires manual connecting to the target cluster
+OpenShift Connector plug-in requires manual connecting to the target cluster
```
Comment (yhontyk) [inline modules/extensions/partials/proc_authenticating-with-openshift-connector-from-che.adoc]:
```suggestion
OpenShift Connector plug-in requires manual connecting to the target cluster.
```

File: modules/extensions/partials/proc_authenticating-with-openshift-connector-from-che.adoc [hunk 5]
```diff
@@ -1,53 +1,63 @@
...
+
+The OpenShift Connector plug-in logs into the cluster as `inClusterUser`. If this user doesn't have manage project permission, this error message appears when creating a project using OpenShift Application Explorer:
```
Comment (yhontyk) [inline modules/extensions/partials/proc_authenticating-with-openshift-connector-from-che.adoc]:
```suggestion
The OpenShift Connector plug-in logs in to the cluster as `inClusterUser`. If this user does not have manage project permission, this error message appears when creating a project using OpenShift Application Explorer:
```

File: modules/extensions/partials/proc_authenticating-with-openshift-connector-from-che.adoc [hunk 6]
```diff
@@ -1,53 +1,63 @@
...
-To work around this temporary issue, log out from the local cluster and relog in to OpenShift cluster using the OpenShift user's credentials.
+To work around this issue:
+
+. Log out from the local cluster.
+. Log in to OpenShift cluster using the OpenShift user's credentials.
 ====
 
-When using a local instance of OpenShift (such as CodeReady Containers or Minishift), the user’s credentials are stored in the workspace `~/.kube/config` file, and may be used for automatic authentication in subsequent logins. In the context of {prod-short}, the `~/.kube/config` is stored as a part of the plug-in sidecar container.
+ifeval::["{project-context}" == "che"]
+When using a local instance of OpenShift such as CodeReady Containers or Minishift, {prod-short} stores the user’s credentials in a `~/.kube/config` file in the workspace. Use this file may for automatic authentication in subsequent logins. In the context of {prod-short}, the `~/.kube/config` is stored as a part of the plug-in sidecar container.
```
Comment (yhontyk) [inline modules/extensions/partials/proc_authenticating-with-openshift-connector-from-che.adoc]:
```suggestion
When using a local instance of OpenShift such as CodeReady Containers or Minishift, {prod-short} stores the user’s credentials in a `~/.kube/config` file in the workspace. Use this file for automatic authentication in subsequent logins. In the context of {prod-short}, the `~/.kube/config` is stored as a part of the plug-in sidecar container.
```

Comment (yhontyk) [reply modules/extensions/partials/proc_authenticating-with-openshift-connector-from-che.adoc]: Or 
Or, alternatively:
```suggestion
When using a local instance of OpenShift such as CodeReady Containers or Minishift, {prod-short} stores the user’s credentials in a `~/.kube/config` file in the workspace. This file may be used for automatic authentication in subsequent logins. In the context of {prod-short}, the `~/.kube/config` is stored as a part of the plug-in sidecar container.
```

File: modules/extensions/partials/proc_authenticating-with-openshift-connector-from-che.adoc [hunk 7]
```diff
@@ -1,53 +1,63 @@
...
 Search for *OpenShift: Log in to cluster* and press kbd:[Enter].
 . If a *You are already logged in a cluster.* message appears, click *Yes*.
+
+. Select the method to log in to the cluster: *Credentials* or *Token*, and follow the login instructions.
 +
-A selection whether to log in using *Credentials* or *Token* is displayed at the top of the screen.
-. Select the method to log in to the cluster and follow the login instructions.
-+
-NOTE: For authenticating with a token, the required token information is in the top right corner of the main {ocp} screen, under *__<User name>__ -> Copy Login Command*.
+NOTE: To authenticate with a token, the required token information is in the top right corner of the main {ocp} screen, under *__<User name>__ > Copy Login Command*.
```
Comment (yhontyk) [inline modules/extensions/partials/proc_authenticating-with-openshift-connector-from-che.adoc]:
```suggestion
NOTE: To authenticate with a token, the required token information is in the upper right corner of the main {ocp} screen, under *__<User name>__ > Copy Login Command*.
```

File: modules/extensions/partials/proc_authenticating-with-openshift-connector-from-che.adoc (hunk 3 above)
Comment (themr0c) [reply modules/extensions/partials/proc_authenticating-with-openshift-connector-from-che.adoc:15]: `that` triggers a SG error...

Comment (yhontyk) [reply modules/extensions/partials/proc_authenticating-with-openshift-connector-from-che.adoc:15]: Damn, ok, let's leave the gerunds then.

**Review by yhontyk — APPROVED** [review]

**Comment by themr0c** [comment]
We need your review @skabashnyuk.

**Review by skabashnyuk — APPROVED** [review]

**Pull request merged**